libraries/AP_Arming/AP_Arming.h

```diff
@@ -43,6 +43,7 @@ class AP_Arming {
         VISION      = (1U << 18),
         FFT         = (1U << 19),
         OSD         = (1U << 20),
+        RALLY       = (1U << 21),
```

Comment:
Do we need a arming bit if were defaulting the threshold to 0 for disabled? We might add other rally checks in the future of course, but we could always add the bit then.